rs/messaging/best-practices-panics.md

```diff
@@ -0,0 +1 @@
+replicated_state/best-practices-panics.md
```

Comment:
I was going to point out that this is not a valid Markdown link, but then I realized it's a symlink (something that GitHub is not very good at making you aware of).

Even so, the symlinks appear to be missing a `../` prefix.

Reply:
Good catch, I updated it.